pkg/csi/service/vanilla/controller_helper.go

```diff
@@ -139,3 +144,52 @@ func convertCnsVolumeType(ctx context.Context, cnsVolumeType string) string {
 	}
 	return volumeType
 }
+
+func (c *controller) getBlockVolumeToHostMap(ctx context.Context,
```

Comment:
Ideally this should be a common function for both Vanilla and WCP with slight changes as this code is repetitive. You can consider this change in your next PR as this is only enhancement.